WORKSPACE

```diff
@@ -1,4 +1,4 @@
-# Copyright 2018 Google LLC
+# Copyright 2020 Google LLC
```

Comment:
It looks like a recent Bazel upgrade broke cos-customizer builds, and these WORKSPACE changes are needed for cos-customizer to compile again. This makes these WORKSPACE changes much higher priority than the rest of the changes in this PR. Can we separate out these WORKSPACE changes into a different PR, and we can submit that?